Changes.md

```diff
@@ -21,7 +21,7 @@ Features
   - Locations can have their Visible Set inclusion or exclusion toggled by clicking within the appropriate column. <kbd>Shift</kbd>-clicking will remove the location and any of its descendants from the Visible Set. When multiple locations are selected, edits to any of the selected locations will affect all selected locations.
 - Light Editor
   - Added a column for toggling a light's mute attribute on and off.
-  - Added a "Remove Edit" command to the right-click menu to easily disable tweaks in Edit Scopes.
+  - Added a "Delete Edit" command to the right-click menu, shortcut <kbd>D</kbd>, to easily disable tweaks in Edit Scopes.
```

Comment:
Delete -> Disable

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/4948/commits/f5e07ea1421f5949ce06f2b0021b73365e5c3428